interface/src/app/signature/enriched_signatures.js

```diff
@@ -0,0 +1,203 @@
+/**
+ * "adage.enrichedSignatures" module.
+ */
+
+angular.module('adage.enrichedSignatures', [
```

Comment:
It probably makes more sense to move this file (and the html template) to `gene` directory, because it is based on a search of genes. What do you think? (I was thinking about it when reading Rene's PR but forgot later.)

Reply:
Ah! I see what you mean. It could definitely go in either place, since this feature examines both genes and signatures. I think about this in two ways: (1) the ideal component arrangement, and (2) pragmatically, what works best?

1. From a component-driven (or object-oriented) perspective, the model for `Gene` is more abstract than the model for `Signature`. There can be no `Signature` without the concept of a `Gene`, but a `Gene` does not depend upon the concept of a `Signature`. This indicates that the logic for finding enriched signatures ideally belongs in the component for `Signature`s. The implementation for a `Gene` should not "know" anything about a `Signature`.

2. From a pragmatic perspective, the code should be found easily by a new programmer (or one of us, a year from now) looking for how to change this feature. Since the page is called "enriched signatures," I'd say it would be easiest to find in the `app/signature` component.

So I think both perspectives point toward keeping it there.